stages/settings.h

```diff
@@ -72,12 +72,38 @@ struct PersistentData {
 //  - b00110000 (0x0c00) (12) ->  quantization scale
 //
 
+// Independent EGs state is 12 bytes per envelope (there is one envelope per
+// channel). Each pair of bytes corresponds to slider and pot values. three of
+// the bytes are not used but are kept as padding to keep the bytes in pairs.
+// - byte 0: delay length
+// - byte 1: -- padding --
+// - byte 2: attack length
+// - byte 3: attack curve
+// - byte 4: hold length
+// - byte 5: -- padding --
+// - byte 6: decay length
+// - byte 7: decay curve
+// - byte 8: sustain level
+// - byte 9: -- padding --
+// - byte 10: release level
+// - byte 11: release curve
+const uint8_t IEG_DELAY_LENGTH = 0;
+const uint8_t IEG_ATTACK_LENGTH = 2;
+const uint8_t IEG_ATTACK_CURVE = 3;
+const uint8_t IEG_HOLD_LENGTH = 4;
+const uint8_t IEG_DECAY_LENGTH = 6;
+const uint8_t IEG_DECAY_CURVE = 7;
+const uint8_t IEG_SUSTAIN_LEVEL = 8;
+const uint8_t IEG_RELEASE_LENGTH = 10;
+const uint8_t IEG_RELEASE_CURVE = 11;
```

Comment:
Any reason not to use an enum here? You could do `enum IEGParam : uint8_t` to keep it 1 byte.